### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by YoungFellow-le**
I updated the README.md file to include the AUR button thingy and to include the dependencies in a way that is clear and pleases the eye :)

### CODE REVIEW

**Comment by YoungFellow-le** [comment]
Should I close this?

**Comment by Zzombiee2361** [comment]
That's not necessary

**Comment by YoungFellow-le** [comment]
Maybe we should add a todo list to the README file...?
We can then make an announcement on reddit or something displaying WALC's features, so we can get more pull requests rather than you doing every thing yourself @Zzombiee2361....
Just thinking out loud.....

**Comment by Zzombiee2361** [comment]
Yeah, getting more pull request is nice. I barely have free time to work on this. But I need to tidy up the code first, it's a mess currently

**Review by Zzombiee2361 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,49 +33,54 @@ Secure your chat from prying eyes 👀! _Warning:_ it's not very safe, but it sh
 and many more...
 
 ## Releases
-[![Get it from linux-APPS](https://i.imgur.com/YGU3qMJ.png)](https://www.linux-apps.com/p/1383431/)
+[![Get it from linux-APPS](https://i.imgur.com/YGU3qMJ.png)](https://www.linux-apps.com/p/1383431/) 
+[![Get it from The AUR](https://user-images.githubusercontent.com/79008923/170873436-821df1d3-b45f-441b-b756-d81c6ac8d474.png)](http://aur.archlinux.org/packages/walc)
 
-To download an AppImage, please goto [Latest Release](https://github.com/WAClient/WALC/releases/latest) Page
+To download an AppImage or a DEB installer file, please goto [Latest Release](https://github.com/WAClient/WALC/releases/latest) Page
```
[the remark this answers is not in the document]
Comment (YoungFellow-le) [reply README.md]: I thought I did.... sorry

**Pull request merged**